packages/frontend/src/app/page.tsx

```diff
@@ -10,7 +10,12 @@ import { Table, TableBody, TableHeader } from '@/common/components/Table'
 
 async function getEntities(): Promise<Entity[]> {
   try {
-    const response: Response = await fetch('http://localhost:3000/api/registry')
+    const response: Response = await fetch(
+      'http://localhost:3000/api/registry',
+      {
+        cache: 'no-cache',
+      },
+    )
     const responseJson: TrustRegistry = await response.json()
```

Comment:
Do we have to annotate it with `TrustRegistry`?

Ideally, we should rely on type inference as much as possible.